packages/odyssey/src/scss/abstracts/_functions.scss

```diff
@@ -72,13 +72,14 @@
 
 $icons: (
   'arrow':'<svg viewBox="0 0 10 12" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M4.29289 0.292893C4.68342 -0.0976311 5.31658 -0.0976311 5.70711 0.292893L9.70711 4.29289C10.0976 4.68342 10.0976 5.31658 9.70711 5.70711C9.31658 6.09763 8.68342 6.09763 8.29289 5.70711L6 3.41421V11C6 11.5523 5.55228 12 5 12C4.44772 12 4 11.5523 4 11V3.41421L1.70711 5.70711C1.31658 6.09763 0.683417 6.09763 0.292893 5.70711C-0.0976311 5.31658 -0.0976311 4.68342 0.292893 4.29289L4.29289 0.292893Z" fill="%%COLOR%%"/></svg>',
+  'anchor': '<svg width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M14.264 9.733a4.8 4.8 0 00-6.787 0l-4.36 4.363a4.8 4.8 0 003.393 8.19 4.768 4.768 0 003.39-1.399l3.6-3.6a.343.343 0 00-.243-.585h-.137a5.769 5.769 0 01-2.194-.426.343.343 0 00-.374.076l-2.588 2.591a2.058 2.058 0 11-2.91-2.91L9.43 11.66a2.057 2.057 0 012.907 0c.541.51 1.385.51 1.926 0 .233-.233.374-.542.398-.87a1.371 1.371 0 00-.398-1.056z" fill="%%COLOR%%"/><path d="M20.877 3.12a4.8 4.8 0 00-6.787 0l-3.596 3.592a.343.343 0 00.25.587h.127a5.759 5.759 0 012.19.428.343.343 0 00.374-.075l2.581-2.578a2.058 2.058 0 112.91 2.91l-3.215 3.212-.027.031-1.125 1.118a2.057 2.057 0 01-2.907 0 1.406 1.406 0 00-1.926 0 1.371 1.371 0 00-.398.877c-.025.393.12.777.398 1.056a4.76 4.76 0 001.37.96c.073.034.145.062.217.092.072.031.147.055.22.083.071.027.147.051.219.072l.202.054c.137.035.274.062.415.086.169.025.34.041.51.048h.26l.207-.024c.075-.003.154-.02.243-.02h.117l.236-.035.11-.02.199-.041h.037a4.799 4.799 0 002.225-1.262l4.364-4.364a4.8 4.8 0 000-6.787z" fill="%%COLOR%%"/></svg>',
```

Comment:
Does this still need to go through SVGO? (the `width/height` should have been stripped if so)

Reply:
Nope, It should have been updated - I ran it through the UI. Will give it another look.

Reply:
Fixed.